index.html

```diff
@@ -691,9 +691,13 @@
 
 <div class="practice">
 
-<span class="practicelab" id="principle-identity-per-context">[=User agents=] should support
-their [=users=]' [=autonomy=] by helping them present their intended [=identity=] to each
-[=context=] that they visit.</span>
+<span class="practicelab" id="principle-identity-per-context">[=User agents=] should help
+their users present the [=identity=] those people want in each [=context=] they find
```

Comment:
```suggestion
their users present the [=identity=] they want in each [=context=] they find
```

Nitpick, but "those people" read to me like the UA and the user had to present the identities of a bunch of other folks.

Reply:
Yeah, I included it because otherwise I wasn't sure that it was clear that "they" referred to the users instead of the user agents. Another option is to make the sentence singular:

"A user agent should help its user present the identity they want ..."

No strong preference here.